dvc/remote/azure.py

```diff
@@ -108,7 +108,7 @@ def list_cache_paths(self):
         return self._list_paths(self.path_info.bucket, self.path_info.path)
 
     def _upload(
-        self, from_file, to_info, name=None, no_progress_bar=False, **_kwargs
+        self, from_file, to_info, name=None, no_progress_bar=None, **_kwargs
```

Comment:
Hm, do we need to adjust these though for this PR?

Reply:
no we don't; just not sure it's worth dealing with now. Argh probably easier to just revert rather than debate. nvm.